swap_meet/vendor.py

```diff
@@ -1,2 +1,81 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        self.item = item
+        self.inventory.append(item)
+        return self.item
```

Comment:
No need to create an attribute for the item. We can return the item as is.

```suggestion
    def add(self, item):
        self.inventory.append(item)
        return self.item
```